### DISCUSSION TITLE
Is there a way to change GET variables in the URL?

### DISCUSSION THREAD

**Discussion by andy-flatratenow** [django-commons/django-unicorn#260] [Q&A] [answered]
I'm coding a calendar at the moment.

I would like to adjust the GET vars in the URL when someone changes the month.

eg: 
mysite.com/calendar/?year=2021&month=**8**

the user clicks "next month" and the url changes to:

mysite.com/calendar/?year=2021&month=**9**

without a page refresh.

Is that possible?

Cheers!

**Comment by adamghill** [comment, marked as answer]
https://www.django-unicorn.com/docs/redirecting/#locationupdate should be able to do what you are looking for! Let me know if you run into any issues or if it would make sense to have a `QueryStringUpdate` to make this a little simpler to implement.

**Reply by andy-flatratenow** [reply]
Thanks Adam! I'll give it a shot!

**Reply by andy-flatratenow** [reply]
Wow that worked perfectly! noice.

And there was no flicker or anything - it refreshes the url - it all just happens! 

Brilliant!
